lib/common/client/middleware/response/raise_error.rb

```diff
@@ -25,21 +25,21 @@ def on_complete(env)
 
           def raise_error!
             if status&.between?(400, 599)
-              raise Common::Exceptions::BackendServiceException.new(service_specific_i18n_key, response_values)
+              raise Common::Exceptions::BackendServiceException.new(service_i18n_key, response_values, @status, @body)
```

Comment:
Is `@status` redundant to `response_values.status`?  Same for `response_values.body['source']` & `response_values.body['detail']`.